### DISCUSSION TITLE
goaccess & gandi.net vhosts (datetime error ?)

### DISCUSSION THREAD

**Discussion by RolandTi** [allinurl/goaccess#2500] [Q&A]
Hello,
i'm trying to make goaccess parsing log format from gandi.net (I'm client from the virtua hosting solution).
I found the base.conf with the line : 
`LogFormat "%V %a %l %u %t (%T s) \"%r\" %>s %b \"%{Referer}i\" \"%{User-agent}i\"" vhost_paas`

So I made this command prompt : goaccess /Volumes/XX/access.log --log-format="%V %a %l %u %t (%T s) \"%r\" %>s %b \"%{Referer}i\" \"%{User-agent}i\"" vhost_paas --datetime-format=%d/%b/%Y:%H:%M:%S %z

But I got `Token '[10/Apr/2023:06:59:08' doesn't match specifier '%t`, but I rebuild exactly the same datetime I'm seeing in acess.log : `XXX.fr 54.36.XXX.XX - - [10/Apr/2023:11:52:08 +0000] (0 s) "GET /viewtopic.php?p=XXX HTTP/1.1"...`

Any idea ? I'm sure I'm not the only user of gandi :)

**Comment by jim-j7071** [comment]
I could not get the cmdline or interactive formatting approaches to parse Gandi's vhost_paas logfiles but I was able to get it working with few edits to  goaccess.conf 
Uncomment these lines

```
time-format %H:%M:%S
date-format %d/%b/%Y
```

Now copy the VCOMBINED format but add (%^) to handle the time-taken-to-serve parameter used by vhost_paas.

`log-format %v %h %^[%d:%t %^] (%^) "%r" %s %b "%R" "%u"`

Regards
 JimJ

**Reply by allinurl** [reply]
@jim-j7071 Thank you for sharing that information. It seems like you have the option to parse the time taken to serve using either `%T`. If you'd like, feel free to provide a sample line from your log and I can take a look.
